### ISSUE TITLE
Add scrubbers

### ISSUE THREAD

**Issue by mwinteringham** [approvals/Approvals.NodeJS#62]
As per conversation with Llewellyn

```
  var dateScrubber = function(data){
      return data.replace(/[0-9]*-[0-9]*-[0-9]*/g, '**scrubbed-name**');
    }

    var multiScrubber = function() {
      var scrubbers = arguments;

      return function(data){
        for (var i = 0; i < scrubbers.length; i++) {
          data = scrubbers[i](data);
        }

        return data
      }
    }

    var verify = function(data, scrubber){
      var json = JSON.stringify(data, null, '  ')

      if(scrubber == null){
        scrubber = function(data) {
          return data
        }
      }

      json = scrubber(json);

      that.verify(json, { reporters: [DiffMergeReporter] });
    }
```

**Comment by isidore** [comment]
link: https://github.com/approvals/ApprovalTests.Net/blob/master/ApprovalTests/Scrubber/ScrubberUtils.cs

**Comment by staxmanade** [comment]
Implemented in https://github.com/approvals/Approvals.NodeJS/pull/66/commits/cd7fe6017a5d00462efedbdb7732b3f831b201f5

## TODO

- [ ] Docs and examples

**Comment by staxmanade** [comment]
@mwinteringham check out the docs for how to use it.

Also I just published `approvals@2.0.0` up to npm

/cc @isidore
